var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -126,7 +179,7 @@ class Trilinos(CMakePackage):
     depends_on('superlu-dist', when='@12.6.2:+superlu-dist')
     depends_on('superlu-dist@develop', when='@develop+superlu-dist')
     depends_on('superlu-dist@xsdk-0.2.0', when='@xsdk-0.2.0+superlu-dist')
-    depends_on('superlu+fpic@4.3', when='+superlu')
+    depends_on('superlu@4.3', when='+superlu')
```

Comment:
Yes, but I can override this default in my `packages.yaml` or on the command line:
```
$ spack install trilinos ^superlu~fpic
```
If the package depends on `superlu+fpic`, it needs to explicitly declare this. This isn't something you should enforce in the superlu package, it's a requirement of trilinos.